### PR TITLE
fix: align ImageCard with detroit updates

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

#55

## Description

Aligns the ImageCard component with the changes in Detroit.

## How Can This Be Tested/Reviewed?

Compare the stories on [dev](https://storybook.bloom.exygy.dev/?path=/story/blocks-image-card--image) to the stories on [this PR](https://deploy-preview-42--storybook-bloom-dev.netlify.app/?path=/story/blocks-image-card--image). This PR includes pulling over the Tooltip component.

While testing in Detroit I noticed that while we used to show a tooltip on the `Verified` tags on the ImageCard we no longer were passing that in. Jesse considers that a regression so once we uptake in Detroit we will re-pass in that tooltip.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have made any corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added or updated stories if new changes are not captured in Storybook
- [x] New and existing unit tests pass locally with my changes
- [ ] I have exported any new pieces added to ui-components
- [ ] I have documented this change in the changelog, and any breaking changes are well described

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Either review the Storybook preview or pull the changes down locally and test that the acceptance criteria is met
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

On merge, squash commits.

### CODE REVIEW

File: src/blocks/ImageCard.scss [hunk 1]
```diff
@@ -18,6 +19,11 @@
   background-color: var(--default-background-color);
   border-radius: var(--border-radius);
 
+  button:focus {
```
Comment (emilyjablonski) [inline src/blocks/ImageCard.scss:22]: Cleaner UI for focus state of multiple images

File: src/blocks/ImageCard.scss [hunk 2]
```diff
@@ -98,9 +104,10 @@
       flex-direction: column;
       align-items: center;
       justify-content: center;
-
+      color: var(--bloom-color-white);
```
Comment (emilyjablonski) [inline src/blocks/ImageCard.scss:107]: Darker background color and white text for the more images tile

File: src/blocks/ImageCard.scss [hunk 3]
```diff
@@ -157,4 +165,22 @@
   --footer-justify: center;
   --modal-border: none;
   --modal-shadow: none;
+  @media (min-width: $screen-md) {
```
Comment (emilyjablonski) [inline src/blocks/ImageCard.scss:168]: Better image alignment when the multiple images modal is open

File: src/blocks/ImageCard.tsx [hunk 4]
```diff
@@ -80,11 +84,7 @@ const ImageCard = (props: ImageCardProps) => {
         />
       )
     })
-    return (
-      <aside className="image-card__status" aria-label={`${props.description || ""} Statuses`}>
-        {statuses}
-      </aside>
-    )
+    return <aside aria-label={`${props.description || ""} Statuses`}>{statuses}</aside>
```
Comment (emilyjablonski) [inline src/blocks/ImageCard.tsx:87]: Removed unused class name

File: src/blocks/ImageCard.tsx [hunk 5]
```diff
@@ -158,20 +158,34 @@ const ImageCard = (props: ImageCardProps) => {
...
-                      fill={tag.iconColor ?? IconFillColors.primary}
-                    />
-                  )}
-                  {tag.text}
-                </Tag>
-              </React.Fragment>
+            const tagContent = (
+              <Tag
+                styleType={tag.styleType || AppearanceStyleType.warning}
+                ariaLabel={
```
Comment (emilyjablonski) [inline src/blocks/ImageCard.tsx:164]: This line doesn't exist in Detroit but is an a11y improvement

File: src/blocks/Tooltip.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+
+      <div className="tooltip__children" data-test-id="tooltip-children" ref={childrenWrapperRef}>
+        {children}
+      </div>
+    </div>
+  )
+}
+
+export { Tooltip as default, Tooltip }
```
Comment (ludtkemorgan) [inline src/blocks/Tooltip.tsx:65]: Do we also want to export this in `index.ts` so it can be used externally? Or do we not want to do that at this point?

**Review by ludtkemorgan — APPROVED** [review]
Looks good! Just one question

File: src/blocks/Tooltip.tsx (hunk 6 above)
Comment (emilyjablonski) [reply src/blocks/Tooltip.tsx:65]: Ah thank you good catch!

**Review by ColinBuyck — APPROVED** [review]
LGTM 🐁 once the scrollable prop use on line 196 is updated to reflect the modal updates!

**Pull request merged**